miio/device.py

```diff
@@ -168,6 +168,13 @@ def _fetch_info(self) -> DeviceInfo:
                 "Unable to request miIO.info from the device"
             ) from ex
 
+    @property
+    def device_id(self):
+        """Return device id (did), if available."""
+        if self._protocol._device_id == b"":
+            self._info = self.info()
```

Comment:
I think this should rather perform only the handshake here, as it will deliver the device id already. Would that be enough?

This would allow an easy way to obtain the device id, even if the token is not known, e.g., for leveraging the cloud data to fetch the token.

Reply:
Yes I think indeed only doing the handshake is enough, I will change that

Reply:
done